frontend/src/screens/TaskEmailFormatPage.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Box, TextField, Typography, Button, Switch, FormControlLabel } from '@mui/material';
+
+const TaskEmailFormatPage: React.FC = (): React.ReactNode => {
+    const [taskOn, setTaskOn] = useState(true);
+    const [team] = useState('Monash Nova');
+    const [recruitmentRound] = useState('Monash Nova Round 1');
+    const [opening] = useState('Events Officer');
+    const [emailBody, setEmailBody] = useState('Task Information');
```

Comment:
for setting states we could use zustand instead? happy to leave be since we're on a crunch tho